### PR TITLE
More coverage to util.py

### PR DESCRIPTION
**Pull request by cbowman0**
Adding a few more tests.  I think I have one spot left I can easily add another check for util.py.  I'll verify with codecov.io output.

### CODE REVIEW

File: webapp/tests/test_util.py [hunk 1]
```diff
@@ -1,22 +1,85 @@
...
         results = [ util.is_local_interface(a) for a in addresses ]
-        self.assertEqual( results, [True, True] )
+        self.assertEqual( results, [True, True, True] )
 
         with self.assertRaises(Exception):
             addresses = ['::1:8080']
             results = [ util.is_local_interface(a) for a in addresses ]
 
+    def test_is_escaped_pattern(self):
+        self.assertEqual( util.is_escaped_pattern('asdf'), False )
```
Comment (JeanFred) [inline webapp/tests/test_util.py]: You probably want to use `self.assertTrue` and `self.assertFalse`.

Comment (cbowman0) [inline webapp/tests/test_util.py]: Thanks for the feedback!

File: webapp/tests/test_util.py [hunk 2]
```diff
@@ -1,22 +1,85 @@
...
+from django.conf import settings
 from graphite import util
 from graphite.wsgi import application  # NOQA makes sure we have a working WSGI app
 
 
 class UtilTest(TestCase):
+
+    # TODO: Determine how to craft the user object
+    #def test_getProfile(self):
+    #    request = {user: {is_authenticated:
```
Comment (obfuscurity) [inline webapp/tests/test_util.py]: Do we want to leave this cruft?

File: webapp/tests/test_util.py [hunk 3]
```diff
@@ -1,22 +1,85 @@
...
+
+    def test_getProfileByUsername(self):
+        self.assertEqual( util.getProfileByUsername('unknown'), None)
+#        self.assertEqual( util.getProfileByUsername('default'), 'default' )
```
Comment (obfuscurity) [inline webapp/tests/test_util.py]: Same here.

**Comment by obfuscurity** [comment]
Good stuff. Mind cleaning up those little bits? I'm 👍 otherwise.

**Comment by cbowman0** [comment]
I don't want to leave those bits in

I was abusing the pull request to get codecov.io to show what code paths are not covered, but I've now worked out how to generate that data locally.  I'll push an update within a day or 2 that includes those checked flushed out or removed.

**Comment by obfuscurity** [comment]
Oh sorry, I jumped the gun there. Carry on. 😄

**Comment by cbowman0** [comment]
Ok.  I finished this round.  Please confirm this is ok to merge.

**Comment by obfuscurity** [comment]
Thanks! 👍

**Comment by JeanFred** [comment]
Looks good to me!

**Pull request merged**